apple/internal/partials/resources.bzl

```diff
@@ -231,6 +258,7 @@ def _resources_partial_impl(
     infoplists = []
 
     locales_requested = _locales_requested(config_vars = platform_prerequisites.config_vars)
+    locales_to_exclude = _locales_to_exclude(config_vars = platform_prerequisites.config_vars)
```

Comment:
Mind calling these variables and the function "locales_excluded" to match the inclusion variable and function name?

Reply:
sounds good!